src/apps/lwaftr/conf.lua

```diff
@@ -81,6 +83,8 @@ local lwaftr_conf_spec = {
       inet_mac=required_at_least_one_of('inet_mac', 'next_hop_ipv4_addr'),
       ipv4_mtu=default(1460),
       ipv6_mtu=default(1500),
+      max_fragments_per_reassembly_packet=default(40),
+      max_ipv6_reassembly_packets=default(40000), -- Just under a gig of memory
```

Comment:
If a lwAFTR conf parameter is assigned a default value, the parameter is always present in the resulting configuration table and that's why the selftest is failing. It's necessary to update the selftest with these two parameters.